src/dos/program_mode.cpp

```diff
@@ -290,14 +293,14 @@ void MODE::AddMessages()
 	        "  [color=light-green]mode[reset] [color=white]80x43[reset]\n"
 	        "  [color=light-green]mode[reset] rate=[color=white]32[reset] delay=[color=white]1[reset]");
 
-	MSG_Add("PROGRAM_MODE_INVALID_COMMAND", "Invalid mode command");
+	MSG_Add("PROGRAM_MODE_INVALID_COMMAND", "Invalid mode command.");
```

Comment:
How about using the generic `SHELL_SYNTAX_ERROR` instead of `PROGRAM_MODE_INVALID_COMMAND`?

Reply:
Yeah I'm fine with that.

Reply:
Done.